docssrc/src/argument_entities.md

```diff
@@ -71,6 +71,49 @@ The `PlayerArgument` class is very similar _(almost identical)_ to `EntitySelect
 >
 > The `PlayerArgument` internally uses the `GameProfile` class from Mojang's authlib, which means that this argument has a slight performance overhead compared to using `EntitySelectorArgument.OnePlayer`
 
+<div class="example">
+
+### Example - Warp command
```

Comment:
Hmm, not sure about the example title. Maybe something like "`PlayerArgument` without entity selectors" would be better?

Reply:
Changed 🎉